silnlp/common/paratext.py

```diff
@@ -10,8 +10,9 @@
 from machine.scripture import VerseRef, book_id_to_number
 from machine.tokenization import NullTokenizer
 
-from .corpus import get_terms_glosses_path, get_terms_metadata_path, get_terms_vrefs_path, load_corpus
-from .environment import SIL_NLP_ENV
+from silnlp.common.corpus import get_terms_glosses_path, get_terms_metadata_path, get_terms_vrefs_path, load_corpus
```

Comment:
We use relative imports everywhere else in the repo.

Reply:
It is easier to test the code with direct imports (just execute the one file in the vsc).  This is a limitation of VSC - and I am requesting a feature enhancement to fix it.  I put the relative imports back.

Reply:
VSC jupyter issue: https://github.com/microsoft/vscode-jupyter/issues/7194